tdrs-backend/reports/.gitignore

```diff
@@ -0,0 +1,3 @@
+*
```

Comment:
The .gitignore files allow us to save the directory into the repo, which is needed to generate the html report. It also ignores files for any reports run locally.